src/server/services/MusicresServices.js

```diff
@@ -50,6 +50,79 @@ class MusicresServices {
     );
     return subjectAreas;
   }
+
+  static async getAnalytics(
```

Comment:
@maxpluez Same comment as in Bruincast, add comments describing the data structure you are making